src/ArrivalTypes.elm

```diff
@@ -0,0 +1,9 @@
+module ArrivalTypes exposing (..)
+
+
+type Msg
+    = ShowArrivalTimes
```

Comment:
Noticing that elsewhere you're putting `=` on the same line as the type being defined, but here you've put it on the next line. Is this just some obscure Elm convention I'm missing? I know there's supposed to be a linter that comes with the language...

Reply:
It's done by the linter. I think when defining a type that isn't an alias this happens.